pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -221,7 +234,7 @@ private boolean endCriteriaReached() {
 
   protected void consumeLoop() {
     final long _endOffset = Long.MAX_VALUE; // No upper limit on Kafka offset
-    segmentLogger.info("Starting consumption loop start offset {}, end offset {}", _currentOffset, _endOffset);
+    segmentLogger.info("Starting consumption loop start offset {}, finalOffset {}", _currentOffset, _finalOffset);
```

Comment:
What do you think about displaying this as a range(eg. Starting consumption loop for offset range [123; 456]) instead of start/end offset?